setup.cfg

```diff
@@ -13,4 +13,32 @@ flake8-ignore = E133 E203 E226 E228 N802 N803 N806 N812 N813 N815 N816 W503
 log_level = DEBUG
 
 [metadata]
+name= ts-cbp
 version = attr: setuptools_scm.get_version
+license=GPL
+project_urls=
+    Bug Tracker=https://jira.lsstcorp.org/secure/Dashboard.jspa
+    Source Code=https://github.com/lsst-ts/ts_CBP
+
+[options]
+package_dir=
+    =python
+packages=find_namespace:
+scripts=
+    bin/run_cbp.py
+tests_require=
```

Comment:
@jonathansick made a comment on lsst/lsst_versions#1 that `tests_requires` is now deprecated and SQuaRE use a "dev" extra: eg https://github.com/lsst-sqre/safir/blob/8930749f687c8f8e135d2926d00722478ba7dd1c/setup.cfg#L51